.github/workflows/build-unimath.yml

```
@@ -5,6 +5,8 @@ on:
     branches: [master]
   pull_request:
     branches: [master]
+    types:
+      - ready_for_review
```

You might also want to add
```suggestion
    types:
      - ready_for_review
      - opened
      - reopened
      - synchronize
```

Due to the `if`condition you add below, this will make it so if a pull request is opened or reopened in ready for review mode, or commits are pushed on a ready-for-review PR, then the CI is restarted.